src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+
+import components from './components';
+
+export default combineReducers({
```

Comment:
Any reason to place the combineReducers() call here when we only have one reducer?

Reply:
Hmm, it was initially put in thinking we'd have more reducers to add in the future, but since we've talked about moving away from redux it makes sense to remove.